src/main/webapp/WEB-INF/cron.xml

```diff
@@ -12,6 +12,12 @@
       <timezone>Australia/Perth</timezone>
     </cron>
     <cron>
+      <url>/feedbackSessionClosedReminders</url>
+      <description>Checks for feedback sessions that are closed and send email to all the instructors of that course.</description>
+      <schedule>every 60 minutes from 00:06 to 23:59</schedule>
+      <timezone>Australia/Perth</timezone>
+    </cron>
+    <cron>
```

Comment:
Yes, that's how we close a session automatically.
You need to find the place where the code closes a session automatically and then add another method call to send an email.